.swiftlint.yml

```diff
@@ -1,6 +1,6 @@
-line_length:
-  warning: 150
-  ignores_comments: true
 
 disabled_rules:
-  identifier_name
+- identifier_name
+- trailing_whitespace
+- line_length
```

Comment:
So if you enable `trailing_whitespace` it throws a warning for blank line between methods. I like to separate each method with one empty line to make it easier to read. That is why i disabled it. 

![screen shot 2017-08-05 at 8 12 26 pm](https://user-images.githubusercontent.com/6182351/29000007-864e54f4-7a1a-11e7-951f-e98d777b42ff.png)